deploy/helm_charts/mixer/templates/deployment.yaml

```diff
@@ -71,13 +71,18 @@ spec:
             requests:
               memory:  {{ .resources.memoryRequest }}
           args:
+            # Import group tables are need for both Mixer and Recon services.
+            - --import_group_tables=$(IMPORT_GROUP_TABLES)
+            {{- if eq $serviceName "recon"}}
+            - --serve_recon_service=true
+            - --serve_mixer_service=false
+            {{ else }}
             - --mixer_project=$(MIXER_PROJECT)
             - --store_project=$(STORE_PROJECT)
             - --bq_dataset=$(BIG_QUERY)
-            - --serve_recon_service=true
```

Comment:
should the recon conditions in its own if else clause? like

```
{{- if eq $serviceName "recon"}}
- --serve_recon_service=true
- --serve_mixer_service=false
{{ else }}
- --serve_recon_service=false
- --serve_mixer_service=true
{{end}}
```

Reply:
yes, that sees much more clear, fixed. Thanks!